src/main/java/Display.java

```diff
@@ -1,13 +1,11 @@
-import java.util.List;
-
 public class Display {
     private ConsoleWriter consoleWriter;
 
     public Display(ConsoleWriter consoleWriter) {
         this.consoleWriter = consoleWriter;
     }
 
-    public void show_grid(Grid grid) {
+    public void grid(Grid grid) {
```

Comment:
Why did you change the naming?

Reply:
`grid` isn't as explicit as I thought it would be. I'll change this. Thanks